Model/Config/Source/YotpoOrderStatusBuilder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Yotpo\Yotpo\Model\Config\Source;
+
+use Magento\Framework\Option\ArrayInterface;
+
+class YotpoOrderStatusBuilder implements ArrayInterface
+{
+    protected $_manager;
+
+    public function __construct(\Magento\Sales\Model\ResourceModel\Order\Status\Collection $manager)
+    {
+        $this->_manager = $manager;
+    }
```

Comment:
You were right, I had just noticed and recalled my memory. It was created unintentionally during debugging the issue caused because of class compilation, I tried to get it done through wrapper but later I figured out the issue was in class compilation because wrapper didn't worked too. I had reverted the wrapper now.